simplified-ui-settings/src/main/java/org/nypl/simplified/ui/settings/SettingsFragmentDebug.kt

```diff
@@ -166,7 +86,33 @@ class SettingsFragmentDebug : Fragment() {
     this.crashlyticsId =
       view.findViewById(R.id.settingsVersionCrashlyticsID)
 
-    return view
+    this.drmTable.addView(
+      this.createDrmSupportRow("Adobe Acs", this.viewModel.adeptSupported)
+    )
+    this.drmTable.addView(
+      this.createDrmSupportRow("AxisNow", this.viewModel.axisNowSupported)
+    )
+
```

Comment:
Nice to see this! This has been something I've been meaning to do for a while. We have quite a few optional DRM components that could be added here, but they don't need to go in right now.